include/envoy/router/router.h

```diff
@@ -204,6 +205,12 @@ class RetryPolicy {
    */
   virtual const std::vector<uint32_t>& retriableStatusCodes() const PURE;
 
+  /**
+   * List of response headers that should trigger a retry when the retriable-headers retry
+   * policy is enabled.
+   */
```

Comment:
Please use the doxygen `@return` tag as in the other methods.

Reply:
Done.